docs/jwst/flatfield/reference_files.rst

```diff
@@ -1,5 +1,5 @@
-Reference File Types
-====================
+Reference File 
+===============
 There are four reference file types for the flat_field step.  Reftype
 FLAT is used for all data except NIRSpec.  NIRSpec data use three
```

Comment:
Should be expanded to "... FLAT is used for all exposure types except NIRSpec spectra", because NIRSpec imaging uses a regular FLAT (not the other 3 types).